converters/md2html_converter.py

```diff
@@ -52,8 +52,6 @@ def convert_obs(self):
                     self.logger.warning(warning)
                 for error in inspector.logger.logs["error"]:
                     self.logger.error(error)
-
```

Comment:
Because the white space got between an if: and else: statement...I normally don't use those there.